templates/zerver/about.html

```diff
@@ -0,0 +1,124 @@
+  drives this is Zulip's unique threading model and system for
+  tracking unread messages, which makes it easy to have multiple
+  simultaneous conversations in the same stream.  As a result, Zulip
+  is more efficient than any other chat product for catching up on
+  conversations you missed while you were away from your devices.  You
+  can read exactly the threads that are important to you, and it feels
+  natural to follow up on conversations that happened while you were
+  away.
+</p>
+
+<p>
+  Zulip's vision is to create the world's best group chat software,
+  completely open source, so that everyone has the freedom to
+  customize, improve, and run their own copy of this essential piece
+  of collaboration infrastructure.
+</p>
+
+<p>
+  Further information on the Zulip project and <a href="/features">its
+  features</a> can be found on
+  the <a href="https://www.zulip.org">Zulip open source project's
+  website</a> and
+  the <a href="http://zulip.readthedocs.io/en/latest/">Zulip
+  documentation</a>.
+</p>
+
+<h2>Zulip Community</h2>
+
+<p>
+  Zulip is developed by a vibrant community of developers from around
+  the world.  Every month, dozens of people contribute code to Zulip,
+  and dozens more contribute bug reports, feedback, and translations.
+  The Zulip community welcomes new contributors from any background.
+  The Zulip project has an easy to install development environment, an
+  extensive test suite,
+  and <a href="http://zulip.readthedocs.io/en/latest/">over 50,000
+  words of developer documentation</a> to make it easy for new
+  contributors to contribute effectively to the project.
+</p>
+
+<h3>Contributing to Zulip</h3>
+
+<p>
+  If you'd like to join the Zulip community and contribute to Zulip,
+  we'd love to have you!  Please
+  visit <a href="https://github.com/zulip/zulip/#zulip-overview">the
+  main Zulip project on GitHub</a> for details on how to get involved!
+</p>
+
+<h2>Zulip history</h2>
+
+<p>
+  Zulip was originally developed by Zulip, Inc., a small startup in
+  Cambridge, Massachusetts.  Zulip, Inc. was founded by the MIT team
+  that previously created
+  <a href="https://www.ksplice.com">Ksplice</a>, software for
+  live-patching a running Linux kernel.  Zulip was inspired by
+  the <a href="https://barnowl.mit.edu/">Barnowl</a> client for
+  the <a href="https://en.wikipedia.org/wiki/Zephyr_(protocol)">Zephyr</a>
+  protocol, and the incredible community that Zephyr supported at MIT.
+</p>
+
+<p>
+  Zulip, Inc. was acquired by Dropbox in early 2014, while the product
+  was still in private beta. Zulip's beta
+  users <a href="https://www.recurse.com/blog/90-zulip-supporting-oss-at-the-recurse-center">loved
+  Zulip's unique user experience</a> and continued
+  using Zulip despite the fact that Zulip was not being actively
+  developed.  After a year and a half, Dropbox generously decided to release Zulip as open
+  source software so that Zulip's users could continue
+  enjoying the software.
```

Comment:
Seven Zulips in three sentences. This is getting stressful to read. How about:

> The corporation was acquired by Dropbox in early 2014, while the product itself was still in private beta. Our beta users <a href="https://www.recurse.com/blog/90-zulip-supporting-oss-at-the-recurse-center">loved Zulip's unique user experience</a> and continued using it even though it was not being actively developed. After a year and a half, Dropbox generously decided to release Zulip as open source software so that users could continue enjoying the software.